x-pack/plugins/security_solution/public/overview/components/detection_response/alerts_by_status/alerts_by_status.tsx

```diff
@@ -80,6 +81,13 @@ interface AlertsByStatusProps {
   signalIndexName: string | null;
 }
 
+enum Statuses {
+  OPEN = 'open',
+  CLOSED = 'closed',
+  ACKNOWLEDGED = 'acknowledged',
+}
+
+const KIBANA_WORKFLOW_STATUS = 'kibana.alert.workflow_status';
```

Comment:
This already exists as `ALERT_WORKFLOW_STATUS` from `'@kbn/rule-data-utils'`

Reply:
changed